html-templates/home.tpl

```diff
@@ -12,8 +12,7 @@
 
         <div class="row">
 
-            <div class="col-12 col-xl-8" style="padding-right:0;">
-
+            <div class="col-12 col-xl-7" style="padding-left:3%; padding-right:3%; ">
```

Comment:
It's best not to hardcode CSS attributes into the HTML in general, and especially not when we're using a responsive framework. 

See https://getbootstrap.com/docs/4.3/utilities/spacing/ for the CSS classes bootstrap provides for adding margin/padding within a grid. Also the `col-xl-*` classes between this and its sibling should add up to 12 per [bootstrap's grid system](https://getbootstrap.com/docs/4.0/layout/grid/)

Be sure to test any layout changes as you resize the window to make sure they work at all the different responsive breakpoints. This column and its sibling are configured to be full-width by default, and then be side-by-side in a 7-to-4 (should be 7-to-5 or 8-to-4 to add up to 12 though) on `xl` screens)

Reply:
Was wondering about the "col-12 col-xl-7" applied to the video section, since there are no breakpoints with the 1 video. Shouldn't this be it's own entity in css ? doesn't make sense to me... will read up on it

WRT the style in html, was following what was already in there.... sorry, no excuse. I can cancel this and fix it tomorrow

Reply:
Yea saw my mistake as soon as I hit enter. Bad day, shouldn't have been near a keyboard.